src/dymaptic.GeoBlazor.Core/Components/Renderers/DimensionalDefinition.cs

```diff
@@ -0,0 +1,53 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Layers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace dymaptic.GeoBlazor.Core.Components.Renderers;
+
+/// <summary>
+///     Defines the symbols to use in a UniqueValueRenderer. Each unique value info defines a symbol that should be used to
+///     represent features with a specific value.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-support-DimensionalDefinition.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public class DimensionalDefinition : LayerObject
+{
+    public DimensionalDefinition() { }
+
+    public DimensionalDefinition(string dimensionName, bool isSlice, string variableName, List<long> values)
+    {
+        DimensionName = dimensionName;
+        IsSlice = isSlice;
+        VariableName = variableName;
+        Values = values;
+    }
+    /// <summary>
+    ///     The dimension associated with the variable..
+    /// </summary>
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? DimensionName { get; set; }
```

Comment:
These properties should be `[Parameter]` in this class

Reply:
corrected